### ISSUE TITLE
No signature of method: liquibase.changelog.DatabaseChangeLog.include() is applicable for argument types

### ISSUE THREAD

**Issue by stanio** [liquibase/liquibase-groovy-dsl#47]
Trying to parse `foo.groovy`:

```groovy
databaseChangeLog(logicalFilePath: 'foo') {
    includeAll path: 'bar/', relativeToChangelogFile: true
}
```

I'm getting:

```
groovy.lang.MissingMethodException: No signature of method: liquibase.changelog.DatabaseChangeLog.include() is applicable for argument types: (java.lang.String, java.lang.Boolean, liquibase.resource.ClassLoaderResourceAccessor, liquibase.ContextExpression, java.lang.Boolean) values: [bar/baz.groovy, true, liquibase.resource.ClassLoaderResourceAccessor(...), ...]
Possible solutions: include(java.lang.String, boolean, liquibase.resource.ResourceAccessor, liquibase.ContextExpression, liquibase.LabelExpression, java.lang.Boolean, boolean)
```

If I remove the `logicalFilePath: 'foo'` attribute it parses fine.  It appears the 3.7.0+ support added in 7bffec0a44350dd3113ecfa7fe039280e7accc02 misses one of the paths: the `DatabaseChangeLogDelegate(Map params, databaseChangeLog)` constructor.  So the same appears to happen also with:

```groovy
databaseChangeLog(context: 'foo') {
    includeAll path: 'bar/', relativeToChangelogFile: true
}
```

**Comment by stevesaliman** [comment]
Thank you for finding this.  I have a fix for it, which I've pushed.  I'll get out a release in the next few days, after I've had the chance to look in to another issue with the parser.

**Comment by stevesaliman** [comment]
I just released version 2.1.2 of the DSL, which fixes this problem.
